Description: Make Lcalc (1.23) build with GCC 4.9 (move default parameter from definition to declaration)
 This patch just fixes the single instance GCC 4.9 currently complains
 about. Clang is more picky, and will need further fixes (including
 moving a few more invalid default parameters in function *definitions*
 into *declarations* instead).
Author: Leif Leonhardy <not.really@online.de>
Origin: http://git.sagemath.org/sage.git/commit/build/pkgs/lcalc/patches/lcalc-1.23_default_parameters_1.patch?h=develop&id=4d49914efab5a0a91f03542fb5848af7d10a1e5e
Bug: http://trac.sagemath.org/ticket/16208
Bug-Debian: http://bugs.debian.org/746867

--- a/include/Ldirichlet_series.h
+++ b/include/Ldirichlet_series.h
@@ -43,7 +43,7 @@
  //XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 template <class ttype>
 Complex L_function <ttype>::
-dirichlet_series(Complex s, long long N=-1)
+dirichlet_series(Complex s, long long N)
 {
     Complex z=0.;
     long long m,n;
--- a/include/L.h
+++ b/include/L.h
@@ -491,7 +491,7 @@
 
     //#include "Ldirichlet_series.h" //for computing Dirichlet series
     Complex partial_dirichlet_series(Complex s, long long N1, long long N2);
-    Complex dirichlet_series(Complex s, long long N);
+    Complex dirichlet_series(Complex s, long long N=-1LL);
 
     //#include "Ltaylor_series.h" //for computing taylor series for Dirichlet series
     //void compute_taylor_series(int N, int K, Complex s_0, Complex *series);
